test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -221,41 +223,22 @@
         (let [response (http-get "http://localhost:10000/hello-proxy/"
                                  {:follow-redirects false
                                   :as :text})]
-          (is (= (:status response) 200))
-          (is (= (:body response) "Hello, World!")))
+          (is (= (:status response) 302)))
         (let [response (http-post "http://localhost:10000/hello-proxy/"
                                  {:follow-redirects false
                                   :as :text})]
-          (is (= (:status response) 200))
-          (is (= (:body response) "Hello, World!")))
+          (is (= (:status response) 302)))
         (let [response (http-get "http://localhost:10000/hello-proxy/world")]
           (is (= (:status response) 200))
           (is (= (:body response) "Hello, World!")))))
 
-    (testing "proxy redirect fails if :follow-redirects set to false"
+    (testing "proxy redirect fails on POST unless :force-redirects set true"
```

Comment:
This `testing` doesn't seem to quite match the test, since this test no longer involves a *failing* redirect.